gobblin-service/src/main/java/org/apache/gobblin/service/modules/spec/JobExecutionPlanDagFactory.java

```diff
@@ -72,6 +72,7 @@ public Dag<JobExecutionPlan> createDag(List<JobExecutionPlan> jobExecutionPlans)
       if (jobName == null) {
         continue;
       }
+      log.info("jobName {} included in jobExecutionPlans {}", jobName, jobExecutionPlans);
```

Comment:
Instead of 3 separate logs I think it's more beneficial to create a list of jobNames, and then output it in a single log with the dagId. The DagId should be available after creating the dag so we can log after the loop.